### PR TITLE
Added form action attribute to thread.php file

### PR DESCRIPTION
**Pull request by rdripley**
This fixes issues #3

### CODE REVIEW

**Comment by rdripley** [comment]
DrDoomsday/forum#2

File: public/thread.php [hunk 1]
```diff
@@ -1,7 +1,7 @@
 <?php require('header.php'); ?>
 
 
-<form>
+<form action="post_reply.php" method="post">
```
Comment (paulstatezny) [inline public/thread.php]: So you kind of made this file into a "post thread" **and** a "post reply" page. Which doesn't really make sense. But either way, this action belongs in the form for posting a reply, which is on line 19.

**Comment by paulstatezny** [comment]
:star:

**Comment by rdripley** [comment]
@paulstatezny this is ready for review.

File: public/thread.php [hunk 2]
```diff
@@ -1,19 +1,16 @@
...
 	Title:
 	<input type="text">
```
Comment (paulstatezny) [inline public/thread.php]: You left this input here from the "make thread" form. It doesn't belong.

Comment (paulstatezny) [inline public/thread.php]: In fact the `<form>` doesn't belong either.

File: public/make_thread.php [hunk 3]
```diff
@@ -0,0 +1,17 @@
...
+
+
+<form>
+	Title:
+	<input type="text">
+	<br>
+
+		<textarea>
+			Enter text here...
+		</textarea>
```
Comment (paulstatezny) [inline public/make_thread.php]: Can we fix the indentation here?

File: public/thread.php [hunk 4]
```diff
@@ -1,19 +1,16 @@
...
-	<input type="submit">
-</form>
+<h1>
+	This is the title of the thread.
+</h1>
 
+<div class="thread-post">
+	This will be where the thread post will go. The first thread post from the thread author will be the first post on the page.
+</div>
 	<br>
```
Comment (paulstatezny) [inline public/thread.php]: I would un-indent this `<br>` too.

File: public/thread.php [hunk 5]
```diff
@@ -1,19 +1,12 @@
...
-		</textarea>
-
-	<br>
-	<input type="submit">
-</form>
-
+<div class="thread-post">
+	This will be where the thread post will go. The first thread post from the thread author will be the first post on the page.
+</div>
 	<br>
```
Comment (paulstatezny) [inline public/thread.php]: I would un-indent this `<br>` too.

**Comment by paulstatezny** [comment]
This PR looks complete -- if you can fix the indentation I'll merge it.

**Comment by rdripley** [comment]
@paulstatezny I fixed the indentation.

**Comment by rdripley** [comment]
@paulstatezny I used your code from jsfiddle. I like the way the format looks. I deleted thread.php and thread.html because we aren't using them for anything (as far as I could tell). I made a file called view_thread.php so I could keep it clear in my head which was viewing threads and which was making a thread. I added the code to the header.php file that applies to make_thread.php and view_thread.php

**Comment by rdripley** [comment]
@paulstatezny I just realized I was pushing ALL changes I was making to this pull request. Sorry about that. How do I push to the master from git bash rather than pushing specifically to this branch?

**Comment by paulstatezny** [comment]
@DrDoomsday Ok sounds good. You're getting more comfortable with this -- good!

No worries about making the changes here. It would have been good to do that in a separate pull request, but since you already pushed the commits we'll just go with it.

**Comment by paulstatezny** [comment]
About the indentation: I think you misunderstood, there were just two `<br>` tags that needed indented one to the right. But it's all good.

I'm going to fix some indentation on this new file and merge. You can look at my commit to see what needed changed.

**Comment by paulstatezny** [comment]
Also, good catch wiring up that "new thread" button.

**Comment by paulstatezny** [comment]
Note, I deleted `public/usethistocheckchanges.html`.

**Comment by paulstatezny** [comment]
Actually, I won't merge this, since I made a change. I'll let you (skim) review my code changes and merge when you're ready.

**Pull request merged**